comment-updater-core/src/main/kotlin/org/jetbrains/research/commentupdater/utils/RefactoringUtils.kt

```diff
@@ -0,0 +1,22 @@
+package org.jetbrains.research.commentupdater.utils
+
+import gr.uom.java.xmi.diff.RenameOperationRefactoring
+import org.refactoringminer.api.Refactoring
+import org.refactoringminer.api.RefactoringType
+
+
+object RefactoringUtils {
+    /**
+     * @return: Map from old method name to new method name
+     */
+    fun extractNameChanges(refactorings: List<Refactoring>): HashMap<String, String> {
+        return hashMapOf(*refactorings.filter {
```

Comment:
what "*" means here?

Reply:
It's spread operator. (Convert list of values to varargs) I didn't find more appropriate way to create hashMap from list.

Reply:
It's spread operator. (Convert list of values to varargs) I didn't find more appropriate way to create hashMap from list.